src/signals/incident-management/components/Label/__test__/Label.test.js

```diff
@@ -5,14 +5,14 @@ import 'jest-styled-components';
 
 import Label from '../';
 
-describe.skip('signals/incident-management/components/Label', () => {
+describe('<Label /', () => {
```

Comment:
Personally, I like the 'signals/incident-management/components/Label' indicator, because it allows me to copy and paste it into my console to run that specific test suite. Certainly where there are components and containers that have the same folder name. But, that's a preference.

Reply:
gedaan